commands/local_server_stop.go

```diff
@@ -92,8 +129,26 @@ var localServerStopCmd = &console.Command{
 		if err := g.Wait(); err != nil {
 			return err
 		}
+	}
 
-		ui.Success(fmt.Sprintf("Stopped %d process(es) successfully", running))
-		return nil
-	},
+	ui.Success(fmt.Sprintf("Stopped %d process(es) successfully", running))
+
+	return nil
+}
+
+func localConfiguredAndRunningProjects() (map[string]*projects.ConfiguredProject, error) {
+	proxyProjects, err := proxy.ToConfiguredProjects()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
```

Comment:
I'm not sure we have to load the proxy projects, using `pid.ToConfiguredProjects()` might be enough here

Reply:
Alright, the logic is way simpler like this.